net/pfSense-pkg-pfBlockerNG/files/usr/local/www/pfblockerng/pfblockerng_alerts.php

```diff
@@ -427,7 +427,12 @@
 // Define common variables and arrays for report tables
 $fields_array	= $pfb_local = $pfb_localsub = $dnsbl_int = $local_hosts = array();
 
-$pfblines	= exec("/usr/local/sbin/clog {$filter_logfile} | {$pfb['grep']} -c ^");
+if (substr(trim(file_get_contents('/etc/version')), 0, 3) > '2.5') {
+	$pfblines = exec("{$pfb['grep']} -c ^ {$filter_logfile}");
```

Comment:
Consider using `dump_log()` instead of running grep or cat directly. This kind of command won't see inside rotated log files, so if a rotate happened recently it wouldn't find much of anything.

https://github.com/pfsense/pfsense/blob/master/src/etc/inc/syslog.inc#L581

Reply:
The code if pfBlockerNG is so far behind that of pfBlockerNG-devel, that I am focusing all efforts on the devel branch, and want to make that the next pfBlockerNG release. I am not currently ready for that, but its in the not to distant future.

This commit is just a band-aid to get the Alerts Tab to function. I would really not want to spend a ton of time to make it perfect, if possible.

Reply:
OK, just so long as you aware that it isn't going to function well and can make users of the package aware of the limitation until it's fully addressed.

Reply:
Yes Thanks!